docs/users.rst

```diff
@@ -390,7 +390,7 @@ not
 go
 ~~
 
-.. warning:: Using this key is deprecated in favor of using Cachito integration
+.. warning:: Using this key is deprecated in favor of using Cachito integration. To switch to Cachito, set the ``remote_source`` key instead. OSBS does not permit users to specify a ``go`` key with a ``remote_source`` key.
```

Comment:
IMO we should use `remote_sources` (plural) https://osbs.readthedocs.io/en/latest/users.html#remote-sources, singular `remote_source` will be deprecated too

Reply:
Gotcha, committed this change.